### DISCUSSION TITLE
[@types/cacache] cacache.rm.entry() missing last optional argument

### DISCUSSION THREAD

**Discussion by limzykenneth** [DefinitelyTyped/DefinitelyTyped#66117] [Issues with a @types package]
The documentation for cacache shown [here](https://github.com/npm/cacache#rm-entry) have the signature of `cacache.rm.entry(cache, key, [opts]) -> Promise` while in the types here it only has `function entry(cachePath: string, key: string): Promise<CacheObject>;`, missing the final `[opts]`

`[opts]` in this case only has one property of `removeFully` of type boolean.

[a comment by an automated account is not shown]

**Comment by ffflorian** [comment]
Hi @limzykenneth, this is due to the fact that the types were written for cacache version 15.0. In this version, the option [didn't exist yet](https://github.com/npm/cacache/tree/f9c677b8b37989a6466372277fb312dc8d48e01f#--cacachermentrycache-key---promise). The option was added in [version 15.2](https://github.com/npm/cacache/blob/main/CHANGELOG.md#1520-2021-05-25) though.

Feel free to open a pull request - I might fix it in the future but right now unfortunately I don't have the time for that.
